galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -261,10 +261,10 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
                 if (seType != null) {
                     switch(seType) {
                         case BUILD:
-                            this.runType = RunType.SharedEnvironmentBuild;
+                            this.runType = RunType.SHAREDENVIRONMENTBUILD;
```

Comment:
Suggest `SHAREDENVIRONMENTBUILD` is changed to `SHARED_ENVIRONMENT_BUILD` for readability.